data/remnant missions.txt

```diff
@@ -493,5 +597,771 @@ mission "Remnant: Technology Available"
+		"reputation:  Drak " += 1
+
+		
+mission "Remnant: Learning Sign 1"
+	name "Learning Remnant Sign"
+	description `Visit each of the Remnant worlds to find ways to learn their sign language.`
+	source "Caelian"
+	stopover "Aventine"
+	destination "Viminal"
+	to offer
+		has "license: Remnant"
+		random < 75
+	on offer
+		conversation
+			`As you make your way through the spaceport, you become increasingly aware of how quiet it is. Looking around, you note that it doesn't seem any less busy than usual, just that fewer people are paying any attention to you. As a result, they have reverted to their normal gestures instead of singing. It feels oddly isolating to realize that there are hundreds of conversations going on around you, and you can't understand a word of it.`
+			choice
+				`	(Ask someone to teach you.)`
+					goto AskSign
+				`	(Think about asking later.)`
+					defer
+				`	(Ignore it.)`
+					decline
+			label AskSign
+			`	Looking around the spaceport, you eventually find someone at what looks remarkably like an information desk. You approach and...`
+			choice
+				`	say "I would like to learn how to talk using signs."`
+					goto "words"
+				`	(You sing a short verse about not understanding people because your hands are still.)`
+					goto "song"
+			label "words"
+			`	She looks at you with a confused expression, her hands making a few quick gestures. After a minute she points at the hand scanner embedded in the desk.`
+				goto "handscan1"
+			label "song"
+			`	She looks surprised, but then recovers and sings a verse about how hands can learn but must be cautious, and gestures at the hand scanner embedded in the desk.`
+				goto "handscan1"
+			label "handscan1"
+			`	As you place your hand on the scanner you feel a quick tingle as it scans your hand, then a pinprick as it extracts a drop of blood. The lady behind the desk looks at her screen. She nods, taps on the screen a few times, then retrieves a datapad from beneath the desk.`
+			`	She hands the pad to you, and sings a brief verse about perseverance and hope.`
+			`	...`
+			`	Back on your ship you fip the pad on. The Datapad appears to be loaded with a number of historic musicals and operas where the meaning of the lyrics has been replicated with gestures as sub-titles. It is interspersed with what appear to be kindergarten level videos slowly explaining gestures and pausing to give you the opportunity to practice.`
+			`	After a few hours a note pops up on the screen, telling you that additional lessons are available from the director's desk on Aventine.`
```

Comment:
"After a few hours, a note pops up on the screen, telling you that additional lessons are available from the director's desk on Aventine."
To
"After a few hours, a note pops up on the screen, _informing_ you that additional lessons are available _at_ the director's desk on Aventine."

Reply:
👍